app/src/componentLibrary/sidebar/navbar/navbar.jsx

```diff
@@ -43,11 +48,15 @@ export const Navbar = ({
       {topSidebarControlItems.map(({ sidebarBlockItem, key, onClick }) => (
         <SidebarButton
           key={key}
-          className={cx('navbar-btn')}
+          className={cx('navbar-btn', getClassName(key, true))}
```

Comment:
Please rewrite this and avoid mystery boolean arguments.

Reply:
Done.